Paper.tex

```diff
@@ -1418,6 +1418,13 @@ \section{Signing Transactions}\label{app:signing}
 %\mathtt{\tiny secp256k1p} &= 2^{256} - 2^{32} - 977\\
 \end{align}
 
+Note that $\mathtt{\small ECDSARECOVER}$ function does not follow the distinction of valid or invalid transactions defined here. $\mathtt{\small ECDSARECOVER}$ returns non-empty results if the following conditions are met (this is relevant in the definition of $\Xi_{\mathtt{ECREC}}$).
```

Comment:
Note that the ECDSARECOVER function.

Reply:
What do you mean?

Reply:
These are proposed edits.

Note that *the* $\mathtt{\small ECDSARECOVER}$ function does not follow the distinction of valid or invalid transactions defined *above*. $\mathtt{\small ECDSARECOVER}$ returns non-empty results if the following conditions are met (this is relevant in the definition of *the precompile* $\Xi_{\mathtt{ECREC}}$).

Reply:
That looks better.  I'll modify my branch.